path_login.go

```diff
@@ -45,14 +45,58 @@ func pathLogin(b *backend) *framework.Path {
 			},
 		},
 		Callbacks: map[logical.Operation]framework.OperationFunc{
-			logical.UpdateOperation: b.pathLoginUpdate,
+			logical.UpdateOperation:      b.pathLoginUpdate,
+			logical.ResolveRoleOperation: b.pathResolveRole,
```

Comment:
Given the comment that "core checks the /login endpoint for role resolution" in your description, will the `logical.ResolveRoleOperation` for `login/<role>` actually ever be hit due to role-based quotas?

Reply:
I don't see any harm in it, was mostly wondering if it would have application outside role-based quotas.

Reply:
Right, makes sense. Yeah my thought was that no application is really targeted at this endpoint now but it seems like it has only potential benefits from being there